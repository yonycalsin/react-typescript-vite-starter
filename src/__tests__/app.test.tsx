import { render, screen } from '@testing-library/react'
import App from 'app'
import { describe, expect, it } from 'vitest'

describe('GIVEN App', () => {
  it('should render the title', () => {
    render(<App />)

    expect(screen.getByText(/react/i)).toMatchSnapshot()
  })

  it('should render the author', () => {
    render(<App />)

    expect(screen.getByText(/yony calsin/i)).toMatchSnapshot()
  })
})
