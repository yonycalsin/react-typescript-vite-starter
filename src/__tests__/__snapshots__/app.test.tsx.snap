// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GIVEN App > should render the author 1`] = `
<a
  href="https://www.yonycalsin.com/"
>
  Yony Calsin
</a>
`;

exports[`GIVEN App > should render the title 1`] = `
<h1>
  React + Typescript + Vite - Starter
</h1>
`;
